### DISCUSSION TITLE
Custom repeats for Bills

### DISCUSSION THREAD

**Discussion by Nakazen** [firefly-iii/firefly-iii#4905] [Firefly III]
Hi James,

Could it be possible to add custom repeats for certain bills?
I have a bill that comes in every 4 months, but I can't reliably get it to trigger on 3 months or 6 months.

Regards,
Naka

**Comment by JC5** [comment]
What do you mean, reliably?

**Reply by Nakazen** [reply]
Because the bill comes in every 4 months it will show an unpaid bill for a month or a bill that comes in 2 months earlier than expected.
This makes the bill overview "unreliable" as I use that to check if all the bill have been successfully paid for that month.

**Reply by JC5** [reply]
But does it say “not expected this period”?

**Comment by JanFeld** [comment]
I would be happy to see this feature too.

There's a bill which i have to pay from March to December but not January and February.
With the current options i can't activate/deactivate the bill automatically.
I have to check and switch it manually.

If one could select a time range based on the current setting (week, month, ..) similar to the custom time-range overview setting, it would make this much easier and more comfortable.

Cheers,
Jan
